documentation/CONTRIBUTE.md

```diff
@@ -1,27 +1,11 @@
-# Debugging (WinDbg)
-
-The following script allows to debug the profiler with `windbg` on Windows 10 with the Windows SDK installed:
-
-    SET COR_ENABLE_PROFILING=1
-    SET COR_PROFILER={DD0A1BB6-11CE-11DD-8EE8-3F9E55D89593}
-    SET COR_PROFILER_PATH=E:\Profiler\bin\Debug\Profiler64.dll
-
-    "C:\Program Files (x86)\Windows Kits\10\Debuggers\x64\windbg.exe" "E:\test-data\test-programs\GeneratedTest.exe"
-
-When `windbg` starts up, hit 'Insert and remove breakpoints' (`F9`) and add a breakpoint to
-
-    bu Profiler64!CProfilerCallback::Initialize
-
-Then hit 'Go' (`F9`).
-The program will stop at the invocation of our initialization and open a source view showing this location.
-From this point you can debug with step over/into/out as usual.
-
-### Notes
-
-* `windbg` normally finds the source files automagically.
-  If it doesn't, setting `File > Source File Path...` probably helps.
-* You may [set more specific breakpoints][breakpoints].
-* Credits to David Broman and his blog post [explaining how to debug profilers using `windbg`][debugProfilers].
-
-  [debugProfilers]: https://blogs.msdn.microsoft.com/davbr/2007/12/11/debugging-your-profiler-i-activation/
-  [breakpoints]: https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/breakpoint-syntax
+# Debugging
+
+The `Sample Debugging App` inside this repository serves to debug the profiler.
+To get it going please follow these steps:
+	* In Visual Studio, select the `Debug` solution config and `x64` as solution platform.
+	* Build the profiler.
+	* In the Sample Debugging App properties go to the `Debug` section and select `Open debug launch settings UI`.
+	* Change `CORECLR_PROFILER_PATH_64` to the path of your `Profiler64.dll`. This path needs to be absolute and should look similar to `C:\\Dev\\teamscale-profiler-dotnet\\Profiler\\bin\\Debug\\Profiler64.dll`.
+	* Start the Sample Debugging App via Visual Studio. 
+	
+Your breakpoints in the profiler should now be working correctly and help you with debugging.
```

Comment:
Could you also add a point on the `Profiler.yml`? 😁 I realize that by creating a fresh git clone it does not exist, but I ran into the issue that I already had a Profiler.yml file from a previous run right next to the Profiler64.dll. This excluded the Sample Debugging App from profiling, causing that nothing happened when I was trying to debug. 😅

Could be something like: "Make sure that if a `Profiler.yml` configuration file exists in the DLLs folder, it should enable profiling for `Sample_Debugging_App.exe`"